packages/config/src/genesisConfig/types.ts

```diff
@@ -1,13 +1,15 @@
 import {ForkName} from "@lodestar/params";
-import {DomainType, ForkDigest, Root, Slot} from "@lodestar/types";
+import {DomainType, Epoch, ForkDigest, Root, Slot} from "@lodestar/types";
+import {BlobScheduleEntry} from "../chainConfig/types.js";
 
 export type ForkDigestHex = string;
 
 export type ForkDigestContext = {
   forkDigest2ForkName(forkDigest: ForkDigest | ForkDigestHex): ForkName;
+  forkDigest2ForkNameWithEpoch(forkDigest: ForkDigest | ForkDigestHex): {fork: ForkName; epoch: Epoch | null};
   forkDigest2ForkNameOption(forkDigest: ForkDigest | ForkDigestHex): ForkName | null;
-  forkName2ForkDigest(forkName: ForkName): ForkDigest;
-  forkName2ForkDigestHex(forkName: ForkName): ForkDigestHex;
+  forkName2ForkDigest(forkName: ForkName, blobSchedule: BlobScheduleEntry | null): ForkDigest;
+  forkName2ForkDigestHex(forkName: ForkName, blobSchedule: BlobScheduleEntry | null): ForkDigestHex;
```

Comment:
with this enhancement this name is not suitable anymore, maybe just go with `getForkDigest(epoch: Epoch): ForkDigest;` instead